package.json

```diff
@@ -66,6 +66,7 @@
     "babel-polyfill": "^6.26.0",
     "bootstrap-sass": "^3.3.7",
     "brace": "^0.10.0",
+    "classnames": "^2.2.6",
```

Comment:
we already use 2.2.5 in our current packaging it seems, if you downgrade it we can avoid updating that.